web_pivot_computed_measure/static/src/test/test.esm.js

```diff
@@ -1,29 +1,18 @@
-/* Copyright 2022 Tecnativa - Carlos Roca
+/* Copyright 2022 Tecnativa - Carlos Roca, Kencove - Mohamed Alkobrosli
```

Comment:
```suggestion
/* Copyright 2022 Tecnativa - Carlos Roca
     Copyright 2025 Kencove - Mohamed Alkobrosli
```
Set the copyright on a different line with correct year